File: esercizi_java/compito_java/Furgone.java
public class Furgone extends Veicolo {
    private int posti;
    private boolean vano;
    private String categoria;

    public Furgone(int cod, String targa, String modello, String marca, double costo, String categoria, int posti,
            boolean vano) {
        super(cod, targa, modello, marca, costo);
        this.posti = posti;
        this.vano = vano;
        this.categoria = categoria;

    }

    @Override
    public String toString() {
        return "commerciale\t" + super.toString() + "\t-\t" + "-\t" + categoria + "\t" + vano;
    }
}

File: esercizi_python/compito_python/furgone.py
from veicolo import veicolo

class furgone(veicolo):
    def __init__(self, cod, targa, modello, marca, cost, categoria, posti, vano):
        super().__init__(cod, targa, modello, marca, cost)
        self.categoria = categoria
        self.posti = posti
        self.vano = vano

    def toString(self):
        return "commerciale" + "\t"+ super().toString() + "\t-"+"\t-"+"\t"+self.categoria+"\t"+str(self.posti)+"\t"+str(self.vano)